packages/smart-wallet/src/smartWallet.js

```diff
@@ -1052,6 +1057,19 @@ export const prepareSmartWallet = (baggage, shared) => {
          * @throws if the seat can't be found or E(seatRef).tryExit() fails.
          */
         async tryExitOffer(offerId) {
+          const { facets } = this;
+          const amts = await facets.payments
+            .tryReclaimingWithdrawnPayments(offerId)
+            .catch(e => {
+              facets.helper.logWalletError(
+                'recovery failed reclaiming payments',
```

Comment:
recovery ? This method says "Exit offer"

Reply:
In Apr 15 discussion with @turadg and @Chris-Hibbert, we agreed to expand `tryExitOffer` to include reclaiming rather than adding a new `tryReclaimOffer` method.